jot/JotTextEditView.m

```diff
@@ -134,9 +134,9 @@ - (instancetype)init
                                                           
                                                           CGFloat centerAboveKeyboard = keyboardRectEnd.origin.y / 2;
                                                           
-                                                          [self.textView mas_makeConstraints:^(MASConstraintMaker *make) {
+                                                          [self.textView mas_updateConstraints:^(MASConstraintMaker *make) {
```

Comment:
Is this a change we made that's specific to our fork of `jot`? Or was this a bug that existed in the `jot` library that we fixed?

Reply:
A bug that we introduced earlier and now fixed. We want to use `updateConstraints` as there's already a constraint set for `self.textView`